tests/test_udf.py

```diff
@@ -380,3 +384,49 @@ def apply_udf_data(data: UdfData):
         mehh_path = Path(data["mehh.__file__"])
         assert not mehh_path.exists()
         assert f"Cleaning up temporary UDF deps at {mehh_path.parent}" in caplog.text
+
+    def test_run_udf_on_vector_data_cube_with_logging(self, tmp_path):
+        repository_root = Path(__file__).parent.parent
+        datacube = openeo.DataCube.load_stac(
+            url=str(repository_root / "docker/local_batch_job/example_stac_catalog/collection.json"),
+            temporal_extent=["2023-06-01", "2023-06-09"],
+        )
+
+        geometries = {
+            "type": "Polygon",
+            "coordinates": [
+                [
+                    [5.07, 51.215],
+                    [5.07, 51.22],
+                    [5.08, 51.22],
+                    [5.08, 51.215],
+                    [5.07, 51.215],
+                ]
+            ],
+        }
+
+        vector_cube = datacube.aggregate_spatial(geometries=geometries, reducer="mean")
```

Comment:
If you just want to test logging from UDF's I don't think you need all this load_stac/local_batch_job/aggregate_spatial setup. Eventually you just want a simple process graph, with "top level" run_udf process.

Ultimately I even think it should be possible to just work with this minimal, no-cubes, process graph:

```json
{
  "runudf1": {
    "process_id": "run_udf",
    "arguments": {
      "data": [1, 2, 3],
      "runtime": "Python",
      "udf": "UDF CODE HERE"
    }
  }
}
```

Reply:
That works too, awesome!